client/__tests__/e2e/e2e.test.ts

```diff
@@ -678,57 +682,66 @@ for (const option of options) {
       /**
        * (thuang): Test is flaky, so we need to retry until it passes
        */
-      await tryUntil(async () => {
-        if (option.withSubset) return;
+      await tryUntil(
+        async () => {
+          if (option.withSubset) return;
 
-        await setup(option, page);
+          await setup(option, page);
 
-        waitUntilNoSkeletonDetected(page);
+          waitUntilNoSkeletonDetected(page);
```

Comment:
this needs an wait, since it returns a promise
```suggestion
          await waitUntilNoSkeletonDetected(page);
```